lib/std/private/strimpl.nim

```diff
@@ -0,0 +1,41 @@
+func toLowerAscii*(c: char): char {.inline.} =
+  if c in {'A'..'Z'}:
+    result = chr(ord(c) + (ord('a') - ord('A')))
+  else:
+    result = c
+
+template cmpIgnoreStyleImpl*(a, b: typed, caseSensitive: static bool = false) =
```

Comment:
If this is exported may be better to put the actual concrete type? `typed` :arrow_right: `string` or whatever.

Reply:
No, it's intended and it's in `private` module.
I need the type parameter to be both string and cstring. I don't want explicit/implicit conversion from string to cstring.